packages/react-components/react-charts-preview/library/src/components/DonutChart/DonutChart.test.tsx

```diff
@@ -0,0 +1,255 @@
+jest.mock('react-dom');
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import * as React from 'react';
+import { chartPointsDC, chartPointsDCElevateMinimums, pointsDC } from '../../utilities/test-data';
+import * as renderer from 'react-test-renderer';
+import { mount, ReactWrapper } from 'enzyme';
```

Comment:
Fluent v9 uses `@testing-library-react`. Enzyme doesn't have official React 17 or 18 support and we should not introduce it into Fluent v9.